pycds/orm/tables.py

```diff
@@ -276,8 +293,15 @@ class Variable(Base):
     network_id = Column(Integer, ForeignKey("meta_network.network_id"))
 
     # Relationships
-    network = relationship("Network", backref=backref("meta_vars", order_by=id))
-    obs = relationship("Obs", backref=backref("meta_vars", order_by=id))
+    network = relationship("Network", back_populates="variables")
+    meta_network = synonym("network")
+    obs = relationship("Obs", order_by="Obs.id", back_populates="variable")
+    observations = synonym("obs")  # Better name
+    obs_raw = synonym("obs")  # To keep backwards compatibility
+    derived_values = relationship(
+        "DerivedValue", order_by="DerivedValue.id", back_populates="variable"
```

Comment:
Likewise.

Reply:
Ditto. And here, there's no reason there couldn't be more than one derived value from a single variable; in fact, I'd expect it.

Reply:
Nvm; I see that you meant `order_by`. Remove.